src/Extractor.php

```diff
@@ -162,24 +163,27 @@ public function extract(): void
     private function getCustomers(): Generator
     {
         $query = [];
-        $query[] = 'SELECT customer.id, customer.descriptive_name, customer.currency_code, customer.time_zone';
-        $query[] = 'FROM customer';
-        if ($this->config->getSince() && $this->config->getUntil()) {
-            $query[] = sprintf(
-                'WHERE segments.date BETWEEN "%s" AND "%s"',
-                $this->config->getSince(),
-                $this->config->getUntil()
-            );
-        }
-        $query[] = 'ORDER BY customer.id';
+        $query[] = 'SELECT '
+            . 'customer_client.id, '
+            . 'customer_client.manager, '
+            . 'customer_client.descriptive_name, '
+            . 'customer_client.currency_code, '
+            . 'customer_client.time_zone';
+
+        $query[] = ' FROM customer_client';
+        $query[] = ' WHERE customer_client.level <= 1';
+        $query[] = ' ORDER BY customer_client.id';
 
         $search = $this->googleAdsClient->getGoogleAdsServiceClient()->search(
             $this->config->getCustomerId(),
             implode(' ', $query)
         );
 
         foreach ($search->iterateAllElements() as $result) {
-            yield $result->getCustomer();
+            /** @var GoogleAdsRow $result */
+            if ($result->getCustomerClient() instanceof CustomerClient && !$result->getCustomerClient()->getManager()) {
```

Comment:
manager account potom musíme přeskočit, protože ten u sebe data nemá (ve starém extractoru se taky nepouštěl)